apps/indexer/lib/indexer/celo/telemetry/instrumentation/indexer.ex

```diff
@@ -0,0 +1,35 @@
+defmodule Indexer.Celo.Telemetry.Instrumentation do
+  @moduledoc "Metrics for the Indexer application"
+
+  alias Explorer.Celo.Telemetry.Instrumentation
+
+  use Instrumentation
+
+  def metrics do
+    [
+      counter("indexer.import.ingested",
```

Comment:
Inconsistent with underscores everywhere else.

Reply:
thanks, this was testing the auto definition functionality.